tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -59,6 +59,24 @@ def sub(a: int, b: int):
     return a - b
 
 
+def sd_generate(
```

Comment:
is this being used anywhere? don't we already define this in `test_tutorials` too?